### PR TITLE
Update Code.py

### PR DESCRIPTION
**Pull request by Imran-ml**
lr_scheduler_type="cosine"
report_to="tensorboard"
evaluation_strategy="epoch"
These parameters are added to customize the learning rate scheduling, specify the reporting destination for training logs, and determine when the model will be evaluated during training.

### CODE REVIEW

**Review by Imran-ml — COMMENTED** [review]
Perfect
